src/TodoListItem.js

```diff
@@ -1,3 +1,10 @@
 import React from 'react'
-let TodoListItem=(props)=><li key={props.toDo.id}>{props.toDo.title}</li>  
+let TodoListItem=(props)=>{
```

Comment:
Same thing here! Deconstruct in the signature like 
```suggestion
let TodoListItem=({toDo})=>{
```

Reply:
Got it. thank you